### DISCUSSION TITLE
Setting expectations, what is the source of truth for CWEs I can expect codeQL to find?

### DISCUSSION THREAD

**Discussion by shay-legit** [github/codeql#17364] [General]
Hi

Is this list below the one I should look at for supported CWEs per language? Can I expect relevant CWE-s for the specific language to be found (in case of a matching issue)? is there somewhere else I should be looking at?

https://codeql.github.com/codeql-query-help/full-cwe/

I am mostly asking after testing a few things with "WebGoat" and not seeing issues that should have been found according to the above table.

Thank you.

**Comment by hvitved** [comment]
Hi

Yes, I believe that is the list of CWEs that we currently support. Note that some CWEs are extremely broad in scope, for example [CWE-200](https://cwe.mitre.org/data/definitions/200.html), where _sensitive information_ can mean a lot of different things, so claiming full support is virtually impossible.

Out of interest, which results were you missing on WebGoat?
